internal/test/e2e/rhp_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/renterd/api"
+	"go.sia.tech/renterd/internal/test"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestRHPForm(t *testing.T) {
```

Comment:
Love how simple this is now

Reply:
Yeah it feels really good. I think in the coming 3-4 months we'll be able to drastically improve things.